docs/quickstart/python.md

```diff
@@ -167,7 +167,7 @@ service definition.
 From the `examples/python/helloworld` directory, run:
 
 ```sh
-$ python -m grpc_tools.protoc -I../../protos --python_out=. --grpc_python_out=. ../../protos/helloworld.proto
+$ python -m grpc_tools.protoc -I../../protos --python_out=. --grpc_python_out=grpc_2_0:. ../../protos/helloworld.proto
```

Comment:
Do you have to run `protoc` separately with `grpc_2_0` to get the non-gRPC code elements? If yes, we need to amend the instructions with a direct `protoc` invocation.

Reply:
No, this is the only command that has to be run. Double-check my work? Locally delete the `_pb2.py` and `_pb2_grpc.py` files, run this command, and verify that the files are wholly regenerated and completely ready for use?

Reply:
Will do now. Give me a couple minutes.